test/openAPI/features/support/data_read_value.js

```diff
@@ -1,82 +1,81 @@
+        uuid: uuid,
+        field: field,
+        ext: ext,
+      })
 );
 
-Then('The user receives the first name of the searched user', async () => {
-  await specDataReadValue.toss();
-  specDataReadValue.response().should.have.status(200);
-  specDataReadValue.response().should.have.jsonLike('John Helmut');
-});
-
-// Scenario: The user does not obtain the first name of the searched user from the Digital Registries database
-Given(
-  'The user wants to search for the first name of the user in the Digital Registries database and the first name of the record is empty',
-  () => (searchedRecord = '3dcad39a-9abb-4552-954d-c62958d44ec9')
+Then(
+  'User receives a response from the GET \\/data\\/\\{registryname}\\/\\{versionnumber}\\/\\{uuid}\\/read-value\\/\\{field}.\\{ext} endpoint',
+  async () => await specDataReadValue.toss()
 );
 
-// "When" is already written in line 25-34
-
 Then(
-  'The user receives a message that the first name of the searched user is empty',
-  async () => {
-    await specDataReadValue.toss();
-    specDataReadValue.response().should.have.status(200);
-    specDataReadValue.response().should.have.jsonLike('');
-  }
+  'The GET \\/data\\/\\{registryname}\\/\\{versionnumber}\\/\\{uuid}\\/read-value\\/\\{field}.\\{ext} endpoint response should be returned in a timely manner 15000ms',
+  () =>
+    specDataReadValue
+      .response()
+      .to.have.responseTimeLessThan(defaultExpectedResponseTime)
 );
 
-// Scenario: The user is unable to obtain the user's first name from the Digital Registries database because the request is invalid
-Given(
-  "The user wants to search for the user's first name in the Digital Registries database",
-  () => (searchedRecord = '3dcad39a-9abb-4552-954d-c62958d44e')
+Then(
+  'The GET \\/data\\/\\{registryname}\\/\\{versionnumber}\\/\\{uuid}\\/read-value\\/\\{field}.\\{ext} endpoint response should have status 200',
+  () => specDataReadValue.response().should.have.status(200)
 );
 
-When(
-  'The user sends an invalid request to get the first name of the searched user from the Digital Registries database',
+Then(
+  'The GET \\/data\\/\\{registryname}\\/\\{versionnumber}\\/\\{uuid}\\/read-value\\/\\{field}.\\{ext} endpoint response should have content-type: application\\/json header',
   () =>
-    specDataReadValue.get(
-      `${baseUrl(searchedRecord, searchedFieldParameter, searchedFieldExt)}`
-    )
+    specDataReadValue
+      .response()
+      .should.have.header(contentTypeHeader.key, contentTypeHeader.value)
 );
 
 Then(
-  "The result of the operation to receive the user's first name is an error",
-  async () => {
-    await specDataReadValue.toss();
-    specDataReadValue.response().should.have.status(400);
-  }
+  'The GET \\/data\\/\\{registryname}\\/\\{versionnumber}\\/\\{uuid}\\/read-value\\/\\{field}.\\{ext} endpoint response should match json schema',
+  () =>
+    chai
+      .expect(specDataReadValue._response.json)
+      .to.be.jsonSchema({ type: 'string' })
```

Comment:
To have as little as possible hard coded data in our .js files I recommend to put the schema into helpers file. Even if it's invoked once. It's better to store it in one place